Tests/Data/ThermoHydroMechanics/Linear/Point_injection/square_1e2.prj

```diff
@@ -0,0 +1,307 @@
+            <value>0</value>
+        </parameter>
+        <parameter>
+            <name>temperature_ic</name>
+            <type>Constant</type>
+            <value>273.15</value>
+        </parameter>
+        <parameter>
+            <name>pressure_bc_left</name>
+            <type>Constant</type>
+            <value>0</value>
+        </parameter>
+        <parameter>
+            <name>temperature_bc_left</name>
+            <type>Constant</type>
+            <value>273.15</value>
+        </parameter>
+        <parameter>
+            <name>temperature_bc_center</name>
+            <type>Constant</type>
+            <value>11500000</value>
+        </parameter>
+        <parameter>
+            <name>rho_fr</name>
+            <type>Constant</type>
+            <value>999.1</value>
+        </parameter>
+        <parameter>
+            <name>mu</name>
+            <type>Constant</type>
+            <value>1e-3</value>
+        </parameter>
+        <parameter>
+            <name>C_f</name>
+            <type>Constant</type>
+            <value>4280</value>
+        </parameter>
+        <parameter>
+            <name>lambda_f</name>
+            <type>Constant</type>
+            <value>0.6</value>
+        </parameter>
+    </parameters>
+    <process_variables>
+        <process_variable>
+            <name>displacement</name>
+            <components>2</components>
+            <order>2</order>
+            <initial_condition>displacement0</initial_condition>
+            <boundary_conditions>
+                <boundary_condition>
+                    <geometrical_set>square_1x1_geometry</geometrical_set>
+                    <geometry>bottom</geometry>
+                    <type>Dirichlet</type>
+                    <component>1</component>
+                    <parameter>dirichlet0</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>square_1x1_geometry</geometrical_set>
+                    <geometry>left</geometry>
+                    <type>Dirichlet</type>
+                    <component>0</component>
+                    <parameter>dirichlet0</parameter>
+                </boundary_condition>
+            </boundary_conditions>
+        </process_variable>
+        <process_variable>
+            <name>pressure</name>
+            <components>1</components>
+            <order>1</order>
+            <initial_condition>pressure_ic</initial_condition>
+            <boundary_conditions>
+                <boundary_condition>
+                   <geometrical_set>square_1x1_geometry</geometrical_set>
+                   <geometry>out</geometry>
+                   <type>Dirichlet</type>
+				   <component>0</component>
+                   <parameter>pressure_bc_left</parameter>
+                </boundary_condition>
+            </boundary_conditions>
+        </process_variable>
+        <process_variable>
+            <name>temperature</name>
```

Comment:
Point source injection must be modeled as a source term.

Reply:
I have tried the point source injection with source term before. It turns out the source term can not be used for the temperature primary variable...

Reply:
Please post your findings here with the project file etc. and a description of what is not working.
For a working example see https://www.opengeosys.org/docs/benchmarks/elliptic/groundwater-flow-dirichlet-nodal-source-term/.

Reply:
The point injection test have been modified to use the new nodal source boundary. The tests and documents are updated accordingly.